WORKSPACE

```diff
@@ -155,7 +155,7 @@ vaticle_typedb_console_artifact()
 load("@vaticle_typedb_common//dependencies/maven:artifacts.bzl", vaticle_typedb_common_artifacts = "artifacts")
 load("@vaticle_typeql//dependencies/maven:artifacts.bzl", vaticle_typeql_artifacts = "artifacts")
 load("@vaticle_typedb_protocol//dependencies/maven:artifacts.bzl", vaticle_typedb_protocol_artifacts = "artifacts")
-load("//dependencies/maven:artifacts.bzl", vaticle_typedb_artifacts = "artifacts")
+load("//dependencies/maven:artifacts.bzl", vaticle_typedb_artifacts = "artifacts", vaticle_typedb_internal_artifacts = "internal_artifacts")
```

Comment:
These should be moved to 'vaticle/artifacts.bzl' again and also call it `vaticle_typedb_vaticle_maven_artifacts`